### DISCUSSION TITLE
怎么申请contributor

### DISCUSSION THREAD

**Discussion by yuedud** [antvis/X6#3155] [General]
如何申请个contributor呢？

**Comment by NewByVector** [comment]
谢谢你的支持，可以先参与一些小的贡献，可以参考[[如何参与到 X6 开源开发](https://github.com/antvis/X6/issues/1958)](https://github.com/antvis/X6/issues/1958)，随后我们会拉你进群，进行一些技术方案的讨论。

**Reply by yuedud** [reply]
已经加了钉钉了，期待进群

**Reply by NewByVector** [reply]
钉钉号是哪个呢？

**Reply by yuedud** [reply]
struggleroue
